### DISCUSSION TITLE
langium can not upgrade to 3.0.0

### DISCUSSION THREAD

**Discussion by zrwsmd** [eclipse-langium/langium#1410] [Q&A]
when i upgrade to 3.0.0,i replace my dependecies from  "langium": "^1.1.0", to  "langium": "~3.0.0",but an error
![image](https://github.com/eclipse-langium/langium/assets/26995040/b2e63abc-c94a-458c-b845-b30e3969bf5c)
i notice in node_modules ,it has changed to 3.0.0,but my project can not apply it

**Comment by cdietrich** [comment]
i assume you need to change your project setup from cjs to esm as langium is esm since version 2.0
maybe you skaffold a new project with the current cli and compare

**Reply by zrwsmd** [reply]
my project is esm

**Reply by cdietrich** [reply]
i propose you try the scaffold thing anyway and compare

**Reply by montymxb** [reply]
@zrwsmd any update on this here? It may also help if you provide a repo that highlights the issue at hand (assuming you can share what you're working on).
